nextjs/src/components/widgets/mensaje-estatus.tsx

```diff
@@ -0,0 +1,87 @@
+import { Capa } from "components/frameMensaje";
+import { TituloMensaje } from "components/titulo_mensaje";
+import { Cuerpo } from "components/cuerpo_mensaje";
+import { styled } from "styles/stitches.config";
+import { FC } from "react";
+import { Buttons } from "components/button/Buttons";
+
+const TituloWrapper = styled("div", {
+  paddingBottom: "2rem",
+  textAlign: "center",
+});
+
+const CuerpoWrapper = styled("div", {
+  paddingLeft: "1.5rem",
+  paddingRight: "1.5rem",
+  paddingTop: "6rem",
+  paddingBottom: "6rem",
+});
+
+const FrameWrapper = styled("div", {
+  // display: "flex",
+  // minHeight: "100%",
+  // alignItems: "center",
+  // justifyContent: "center",
+  // paddingTop: "3rem",
+  // paddingBottom: "3rem",
+  // paddingLeft: "1rem",
+  // paddingRight: "1rem",
+  // backgroundColor: "rgb(241 245 249)"
+});
+
+interface Props {
+  titulo1: string;
+  titulo2: string;
+  titulo3: string;
+  cuerpo_p1: string;
+  cuerpo_p2: string;
+}
+
+const MensajeEstatus: FC<Props> = ({
+  titulo1,
+  titulo2,
+  titulo3,
+  cuerpo_p1,
```

Comment:
Apegarse a camelCase para los props

```suggestion
  cuerpoParrafoArriba,
```